.github/workflows/build-publish-develop-pr.yml

```diff
@@ -83,12 +83,28 @@ jobs:
             goarch: arm64
             dist_name: linux_arm64_v8.0
     steps:
-      - name: Checkout repository
+      - name: Setup Github Token
```

Comment:
nit: i'd shift this down one step just so readers know that it's for the capabilities registry

Reply:
Done!